src/main/resources/archetype-resources/silo/src/main/java/silo/configuration/LogConfiguration.java

```diff
@@ -29,9 +31,12 @@ private void configureLog4j() throws IOException
     {
         for (String profile : env.getActiveProfiles()) {
             ClassPathResource resource = new ClassPathResource(profile + "/log4j.properties");
-            Properties properties = new Properties();
-            properties.load(resource.getInputStream());
-            PropertyConfigurator.configure(properties);
+
+            try (InputStream configFile = resource.getInputStream()) {
+                PropertyConfigurator.configure(properties);
+                return;
+            }catch (IOException ignore) {
+            }
```

Comment:
If we want it to be as robust as possible there should be a continue so that in case of an exception another profile gets loaded.